src/database-layer/db.ts

```diff
@@ -686,12 +697,14 @@ if (maybePg != null) {
 			transaction: createTransaction(async (stackTraceErr) => {
 				const client = await pool.connect();
 				const tx = new PostgresTx(client, false, stackTraceErr);
+				// eslint-disable-next-line @typescript-eslint/no-floating-promises
 				tx.executeSql('START TRANSACTION;');
 				return tx;
 			}),
 			readTransaction: createTransaction(async (stackTraceErr) => {
 				const client = await replica.connect();
 				const tx = new PostgresTx(client, false, stackTraceErr);
+				// eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
@Page- wouldn't it be better if all the `tx.executeSql` in the file had an  `await`?
Or at least the `START TRANSACTION` ones?